umbral/pre.py

```diff
@@ -524,22 +526,21 @@ def decrypt(ciphertext: bytes, capsule: Capsule, decrypting_key: UmbralPrivateKe
     and return the resulting cleartext.
     """
 
+    if not isinstance(ciphertext, bytes) or len(ciphertext) < DEM_NONCE_SIZE:
+        raise ValueError("Input ciphertext must be a bytes object of length >= {}".format(DEM_NONCE_SIZE))
```

Comment:
I think I would leave out the check to see if it is `bytes` and only check length.

Also, just nitpicking, but change "bytes object" to "bytestring", but it makes sense as it is because Python. :P